### PR TITLE
Support workspace symbols request

### PR DESCRIPTION
**Pull request by datho7561**
Requires https://github.com/eclipse/lsp4mp/pull/313

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
Please add a demo and a section with workspace symbols in https://github.com/redhat-developer/vscode-microprofile/blob/master/docs/JavaFeatures.md

**Comment by datho7561** [comment]
Okay. I've added some documentation. Let me know what you think.

File: README.md [hunk 1]
```diff
@@ -30,38 +30,12 @@ In `microprofile-config.properties` files, you will benefit with:
...
-           *  `@Parameters`
-           *  `@Parameter`
-           *  `@APIResponses`
-           *  `@APIResponse`
+  * [Completion support for MicroProfile](./docs/JavaFeatures.md#completion-support)
+  * [Hover support for MicroProfile](./docs/JavaFeatures.md#hover-support)
+  * [Validation and Quick Fix support for MicroProfile](./docs/JavaFeatures.md#validation-and-quick-fix-support)
+  * [Code Lens support for MicroProfile](./docs/JavaFeatures.md#codelens-support)
+  * [Workspace symbols](./docs/JavaFeatures.md#workspace-symbols-support)
+  * [Code snippets](./docs/JavaFeatures.md#)
```
Comment (angelozerr) [inline README.md:38]: I think it should be

`JavaFeatures.md#snippets-support`

File: docs/JavaFeatures.md [hunk 2]
```diff
@@ -37,3 +37,51 @@ Diagnostics are supported for MicroProfile annotations which targets errors and
 On debug, a Quarkus application run with MicroProfile sources supports a URL CodeLens that routes to the endpoint debug path.
 
 ![MP Java Code Lens](./res/MPJavaCodeLens.png)
+
+## Snippets support
```
Comment (angelozerr) [inline docs/JavaFeatures.md:41]: I will move this section just after completion support

**Pull request merged**